sdk/python/pulumi_kubernetes/apiextensions/CustomResource.py

```diff
@@ -0,0 +1,49 @@
+# *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import pulumi
+import pulumi.runtime
+
+from .. import tables
+
+
+class CustomResource(pulumi.CustomResource):
```

Comment:
is there a reason why we have dupe code here in this file`sdk/python/pulumi_kubernetes/apiextensions/CustomResource.py`, when we define the same code above in `pkg/gen/python-templates/CustomResource.py` ?

Reply:
The copy in `pkg/gen` is copied into the SDK on each build. There are a few exceptions in the NodeJS package I'm intending to update to match, but for the most part, everything in the SDKs is generated to avoid errors as we pull in new OpenAPI schemas from k8s.